app/views/directives/bind-service/delete-binding-result.html

```diff
@@ -0,0 +1,31 @@
+<div>
+  <div ng-if="!ctrl.error">
+    <h3 class="mar-top-none">
+      Binding for the following has been deleted:
+    </h3>
+
+    <div
+      ng-if="ctrl.appsForBinding(ctrl.selectedBinding)"
+      ng-repeat="appForBinding in ctrl.appsForBinding(ctrl.selectedBinding)">
+      {{appForBinding.metadata.name}} <small class="text-muted">&ndash; {{ appForBinding.kind | humanizeKind : true}}</small>
+    </div>
+    <div ng-if="!(ctrl.appsForBinding(ctrl.selectedBinding))">
+      {{ctrl.selectedBinding}} <small class="text-muted">&ndash; Secret</small>
```

Comment:
I think you can put `Binding` here for now and we can circle back. I have some concerns that we don't show the binding name anywhere in the UI, but we can fix as a follow on to get the basic functionality in.